library/x509_crt.c

```diff
@@ -653,12 +724,155 @@ static int x509_get_subject_alt_name( unsigned char **p,
                     MBEDTLS_ERR_ASN1_UNEXPECTED_TAG );
         }
 
-        /* Skip everything but DNS name */
-        if( tag != ( MBEDTLS_ASN1_CONTEXT_SPECIFIC | 2 ) )
+        tag_mask = tag & ( MBEDTLS_ASN1_TAG_CLASS_MASK | MBEDTLS_ASN1_TAG_VALUE_MASK );
+
+        switch( tag_mask )
         {
+        case( MBEDTLS_ASN1_CONTEXT_SPECIFIC | 0 ):
```

Comment:
The indentation is off here.

Reply:
This is same indentation as in the `switch case` in `x509_get_crt_ext()`.
I took the example from that function. Am I missing something?